nixos/modules/services/networking/wireguard.nix

```diff
@@ -45,7 +45,7 @@ let
 
       privateKeyFile = mkOption {
         example = "/private/wireguard_key";
-        type = with types; nullOr str;
+        type = with types; nullOr path;
```

Comment:
This should be a string or path

Reply:
I followed the pattern of the module. Seems other file options also uses path.

Reply:
Right, path is a special string, not the nix path.